### DISCUSSION TITLE
Achievements don't come

### DISCUSSION THREAD

**Discussion by morington** [community/community#106359] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Good day to you all!

Maybe I'm misunderstanding achievements on GitHub, but if I refer to the list of achievements and see what I can get an item for, for some reason I don't get an achievement after completing a task...

Maybe I need to wait for some time?

For example this week I completed the tasks for "Heart On Your Sleeve", "Quickdraw", "Pair Extraordinaire".
and my co-author didn't get "YOLO." I have a suspicion that the achievements are off...(

**Comment by gil-son** [comment, marked as answer]
Hello, @morington 

that [document](https://github.com/4xmen/Get-Github-Achievements/tree/main) can support you

make sure to verify the rules

**Comment by ghostinhershell** [comment]
Hi @morington, thanks so much for posting in our Community. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by morington** [reply]
It's very sad, but I understand.
